SU-HSDO/suhumsci
# STN-580  Vertical Date stacked cards icons

# READY FOR REVIEW

## Summary
Refinement: Adding the same icons for location, day/time, speaker etc as Horizontal date stacked card

Example of Horizontal Cards:
http://sparkbox-sandbox.suhumsci.loc/qa/date-stacked-horizontal-card

JIRA CARD: https://sparkbox.atlassian.net/browse/STN-580

## Need Review By (Date)
asap

## Urgency
medium

## Steps to Test
1. npm run test
2. Verify the icons are not showing up on the Vertical Date Stacked Cards on the home page under "This week's events, seminars and graduate student workshops"
2. Go to the Appearance > Settings and turn on the experimental toggle.
3. Verify the classes for` hb-card__icon` and `hb-card__icon--location`, as well as the date and speaker classes that appear on the cards once the experimental class is turned on.
4. Verify the icons are the correct color pairings for each theme.
5. Verify the classes go away again when the experimental setting is turned off.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
